### PR TITLE
Adding a basic CSS library structure

### PR DESCRIPTION
**Pull request by xeho91**
I woke up early today, so I wanted to be productive by contributing to our work.
I did the research for some quick tasks on our Jira backlog, which can be done quickly, and possibly related to the tasks on current sprint. Also I wanted to test if this is going to work well with current webpack configuration. So far, no problem. 👍
I hope this will "push" and "inspire" to move forward with building our own CSS library.

**Keep in mind, that all of the variables I've put in Library is just an example**.

### CODE REVIEW

File: source/styles/index.css [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+@import "library/fonts.css";
+
 /* Basic styling */
 * {
 	box-sizing: border-box;
 	padding: 0;
 	margin: 0;
 }
 body {
+	background-color: var(--color-primary);
```
Comment (xeho91) [inline source/styles/index.css:14]: @lax0248917

**Comment by xeho91** [comment]
> 
> 
> Sorry this is a couple days late. But this all looks good. Do you think you could add a code snippet as an example of how someone would use these stylings in their HTML. I am assuming that is what we are trying to do, correct?

Library files are imported in `index.css`, which is a "barebone" for the entire website.

All you need to do is just use the correct CSS variable. That's it. Does that answer your question or do you want me to provide an example on some component?

**Comment by lax0248917** [comment]
So if I want to use the CSS library, do I reference the CSS variable in another .CSS file specific for that component? Or will it be more like bootstrap in that if I want a button to be styled a certain way I can use something like 

`<button class="btn-primary">Submit</button>`
where the btn-primary references some certain stylings in our library. Or am I misunderstanding the concepts. Thank you for clearing this up for me, and anyone else who has not done this before. 🚀

**Comment by xeho91** [comment]
> 
> 
> So if I want to use the CSS library, do I reference the CSS variable in another .CSS file specific for that component? Or will it be more like bootstrap in that if I want a button to be styled a certain way I can use something like
> 
> `<button class="btn-primary">Submit</button>`
> where the btn-primary references some certain stylings in our library. Or am I misunderstanding the concepts. Thank you for clearing this up for me, and anyone else who has not done this before. 🚀

If you're sure that the styles for this component will be reusable, then it would be right to put it in the library. I believe.
In the beginning, I imaged we would put mostly variables inside the library, however making the approach to avoid **DRY** pattern _(**D**on't **R**epeat **Y**ourself),_ would probably be helpful. I have no experience it. In my small project, to have more control over the styling of my elements, I kind of ignored this advice and repeated some styles.

Given an example: we want to specify a class `.centered-flex`, which will just centre the items for us (with Flexbox).
The CSS code for this `.centered` class, lets say... inside `library/positioning.css`:
```
.centered-flex {
    display: flex;
    place-items: center;
}
```
On the second thought, at the beginning is probably going to be hard to think/see which specific styles are repeating itself. We can do refactoring/optimizations during the process and focus more on defining variables in the library _(at least those which will be reusable)_.

**Comment by lax0248917** [comment]
Perfect, this clears it up for me quite a bit. Thank you.

File: source/styles/index.css [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
 	font-family: Arial, Helvetica, sans-serif;
```
Comment (mariushilleke) [inline source/styles/index.css:15]: Can we use `font-family: var(--font-sans-serif)`?

Comment (xeho91) [reply source/styles/index.css:15]: Absolutely yes. If we defined (I did) the variable like this.

File: source/styles/index.css [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
 /* Basic styling */
 * {
 	box-sizing: border-box;
 	padding: 0;
 	margin: 0;
 }
 body {
-	font-family: Arial, Helvetica, sans-serif;
+	background-color: var(--color-primary);
+	font-family: var(--font-sans-serif);
```
Comment (mariushilleke) [inline source/styles/index.css:15]: That was the new change.

Comment (xeho91) [reply source/styles/index.css:15]: Yeah, GitHub has a way to show the new changes. 👍 

This is working for you, right? For me it does work.

**Pull request merged**